src/intrinsics/ecma262/ArrayPrototype.js

```diff
@@ -40,6 +40,7 @@ import {
 } from "../../methods/index.js";
 import { Create, Join, Properties, To } from "../../singletons.js";
 import { createOperationDescriptor } from "../../utils/generator.js";
+import NativeFunctionValue from "../../../lib/values/NativeFunctionValue.js";
```

Comment:
You're importing the `lib` value here, you most definitely don't want to do this (VSCode auto does it sometimes).

Reply:
Yikes. Good catch. It was auto-added by VSCode. Perhaps we need a lint rule for this.